src/OgGroupAudienceHelper.php

```diff
@@ -24,6 +24,15 @@ class OgGroupAudienceHelper {
    */
   const DEFAULT_FIELD = 'og_group_ref';
 
+  /**
+   * The name of the field which reference user to groups.
+   */
+  const USER_REFERENCE_FIELD = 'og_membership_reference';
```

Comment:
Typically when defining multiple variations of related constants, the 'fixed' part is put in front, so this is easier to find with autocompletion in the IDE. So it would be better to call it `REFERENCE_FIELD_USER`.

Some examples from core:
* `SAVED_NEW`, `SAVED_UPDATED`, `SAVED_DELETED`, ...
* `ERROR_REPORTING_HIDE`, `ERROR_REPORTING_DISPLAY_ALL`, ...
* `TRANSLATION_REMOVED`, `TRANSLATION_EXISTING`, `TRANSLATION_CREATED`, ...
* etc etc